docs/synapse/userguides/storm_ref_pivot.rstorm

```diff
@@ -7,1027 +7,1721 @@
+.. storm-pre:: inet:fqdn <( ( refs, seen ) )- *
 ::
 
-    <inet:fqdn> -> inet:dns:a:fqdn
+    <inet:fqdn> <( ( refs, seen ) )- *
 
 
-**Implicit syntax:**
+.. _walk-all-edges:
 
+Traverse All Edges
+++++++++++++++++++
 
-.. storm-pre:: inet:fqdn -> inet:dns:a
-::
+Use the wildcard (asterisk) character ( ``*`` ) to traverse any edges present in the specified direction to their targets.
 
-    <inet:fqdn> -> inet:dns:a
+.. TIP::
+  
+  Using the wildcard to specify any edge name is useful when:
+  
+  - you want to traverse any / all edges;
+  - you want to navigate to a specific set of targets (regardless of the particular edges); or
+  - if you are not familiar with the edges that may be used with your source nodes and simply want to explore any
+    connections that may be present.
 
 
-With implicit syntax, the target property ``:fqdn`` can be omitted because it is the only logical target given ``inet:fqdn`` nodes as the source and ``inet:dns:a`` nodes as the target of the pivot.
+**Syntax:**
 
-.. NOTE::
+*<query>* **-(** ***** **)>** *<target>*
 
-  While the ``inet:fqdn`` form has secondary properties that are also of type ``inet:fqdn`` (e.g., both ``:domain`` and ``:zone``) implicit syntax can only be used to pivot between primary and secondary OR secondary and primary properties, but not both. That is, implicit syntax does not allow you to go from any / all properties of a given type in the source nodes to  any / all properties with the same type and value in the target nodes.
-  Because the target of the pivot is ``inet:dns:a`` nodes, the only logical target given the inbound nodes is the ``:fqdn`` **secondary** property, which means the only logical source is the **primary** property of the ``inet:fqdn``.
+*<query>* **<(** ***** **)-** *<target>*
 
-*Pivot from secondary property (<prop> = <pval>) to primary property (<form> = <valu>) using implicit syntax:*
+**Examples:**
 
-- Pivot from a set of DNS A records to their associated IP addresses:
+*For a threat cluster (risk:threat node), traverse any light edges linking the cluster to any vulnerabilities (risk:vuln nodes) or victim organizations (ou:org nodes):*
 
-**Explicit syntax:**
+.. storm-pre:: risk:threat -(*)> ( risk:vuln, ou:org )
 
+::
+  
+  <risk:threat> -(*)> ( risk:vuln, ou:org )
+
+*For a vulnerability (risk:vuln node), navigate to any forms that are connected to the vulnerability by any edge:*
+
+.. storm-pre:: [ risk:vuln=* ] [ <(has)+ { [ it:host=* ] } ] [ <(uses)+ { risk:threat:org:name='vicious wombat' } ]
+.. storm-pre:: risk:vuln <(*)- *
 
-.. storm-pre:: inet:dns:a :ipv4 -> inet:ipv4
 ::
+  
+  risk:vuln <(*)- *
 
-    <inet:dns:a> :ipv4 -> inet:ipv4
+Vertex Project conventions can link ``risk:vuln`` nodes to things that have the vulnerability ( via ``<(has)-``
+edges) or things that use the vulnerability ( via ``<(uses)-`` edges - e.g., a threat cluster (``risk:threat``),
+a file (``file:bytes``) that makes use of an exploit for the vulnerability, etc.
```

Comment:
Missing a closing paren that I think should be at the end of the line:
```suggestion
Vertex Project conventions can link ``risk:vuln`` nodes to things that have the vulnerability ( via ``<(has)-``
edges) or things that use the vulnerability ( via ``<(uses)-`` edges - e.g., a threat cluster (``risk:threat``),
a file (``file:bytes``) that makes use of an exploit for the vulnerability, etc.)
```

Reply:
Cleaned up